elide-datastore/elide-datastore-multiplex/src/main/java/com/yahoo/elide/datastores/multiplex/MultiplexWriteTransaction.java

```diff
@@ -191,44 +187,20 @@ public Object loadObject(Class<?> entityClass,
     }
 
     @Override
-    public <T> Object getRelation(
-            Object entity,
-            RelationshipType relationshipType,
-            String relationName,
-            Class<T> relationClass,
-            EntityDictionary dictionary,
-            Set<Predicate> filters
-    ) {
-        DataStoreTransaction transaction = getTransaction(entity.getClass());
-        Object relation = transaction
-                .getRelation(entity, relationshipType, relationName, relationClass, dictionary, filters);
+    public Object getRelation(DataStoreTransaction relationTx,
+                              Object entity, String relationName,
+                              Optional<FilterExpression> filterExpression,
+                              Optional<Sorting> sorting,
+                              Optional<Pagination> pagination,
+                              RequestScope scope) {
+        Object relation = relationTx
+                .getRelation(relationTx, entity, relationName, filterExpression, sorting, pagination, scope);
```

Comment:
This should work like MultiplexTransaction.getRelation.  I'm confused why this is even overridden.  Is it because of the hold?